src/VirtualGrid.vue

```diff
@@ -422,7 +422,7 @@ export default class VirtualGrid<P> extends Vue {
                 v-for="item in renderData.cellsToRender"
                 :key="item.id"
                 :style="{
-                    'height': item.height,
+                    'height': item.height + 'px',
```

Comment:
Nice catch, I'd suggest this format if I may:

```suggestion
                    'height': `${item.height}px`,
```

Reply:
Updated to use string interpolation.